ui/app/AppLayouts/Chat/components/PrivateChatPopup.qml

```diff
@@ -51,14 +51,14 @@ ModalPopup {
         }
         contentItem: Item {
             anchors.fill: parent
-            SVGImage {
+            RoundedImage {
                 id: dollarEmoji
```

Comment:
nit: probably rename to something else, it is not dollar emoji anymore :dollar: emoji